src/test/kotlin/graphql/kickstart/tools/MissingFieldResolverTest.kt

```diff
@@ -65,9 +67,63 @@ class MissingFieldResolverTest {
         assertEquals(result.getData(), expected)
     }
 
+    @Test
+    fun `should call missing resolver data fetcher provider if provided`() {
+        val missingResolverDataFetcherProvider = TestMissingResolverDataFetcherProvider();
+        val options = SchemaParserOptions.newOptions()
```

Comment:
Let's inline this variable in the builder.

Reply:
It is used later to validate it is passed to the provider when invoked.